modules/compiler/vecz/source/transform/packetization_helpers.cpp

```diff
@@ -253,9 +253,10 @@ Value *createMaybeVPTargetReduction(IRBuilderBase &B,
                                     const TargetTransformInfo &TTI, Value *Val,
                                     RecurKind Kind, Value *VL) {
   assert(isa<VectorType>(Val->getType()) && "Must be vector type");
+  (void)TTI;
```

Comment:
Do we need this?

Reply:
Without this, we may get warnings about the unused parameter depending on the compiler. Casting to `void` is the LLVM preferred style to avoid that, to indicate that it is intentional.

Reply:
But it is being passed to `multi_llvm::createSimpleTargetReduction` below?

Reply:
This is an explicit way of marking a value as unused. I know I could just not give it a name in the signature, but I think this is better because it makes it obvious that it isn't being forwarded to the call.

Reply:
See conversation, I thought the same thing :)

Reply:
Ah, I see now. Sorry for missing that. I also didn't see the conversation because I assumed that Github would poll for new comments every few seconds.

Reply:
Yeah there's a lot more forced refreshing than I'm used to on GitLab.